src/fparser/two/Fortran2003.py

```diff
@@ -9244,8 +9244,8 @@ class containing the name of the program.
     def get_name(self):
         '''Provides the program name as an instance of the `Name` class.
 
-        :returns: the program name as a `Name` class
-        :rtype: `Name`
+        :returns: the program name as a :py:class:`Name` class
```

Comment:
There's no need to duplicate the type specification in the :returns: field but it's not a biggie.